KarhooUISDK/Screens/AddressScreen/AddressMapView/KarhooAddressDisplayView.swift

```diff
@@ -65,8 +65,8 @@ final class KarhooAddressDisplayView: UIView {
     
     func set(addressType: AddressType) {
         switch addressType {
-        case .pickup: addressTypeImage.image = UIImage.uisdkImage("pickup_icon")
-        case .destination: addressTypeImage.image = UIImage.uisdkImage("drop_off_icon")
+        case .pickup: addressTypeImage.image = UIImage.uisdkImage("pickup_icon").coloured(withTint: KarhooUI.colors.primary)
+        case .destination: addressTypeImage.image = UIImage.uisdkImage("pickup_icon").coloured(withTint: KarhooUI.colors.secondary)
```

Comment:
These are some pretty long lines. Maybe have the case on one line and the image setting on the next? Valid for both cases